log

```diff
@@ -0,0 +1,343 @@
+		+   autoscaling_group_name = "${aws_autoscaling_group.bastions-bastions-privateweave-example-com.id}"
+		-   autoscaling_group_name = "${aws_autoscaling_group.bastion-privateweave-example-com.id}"
+		  }
+		  
+		...
+		  }
+		  
+		+ resource "aws_autoscaling_group" "bastions-bastions-privateweave-example-com" {
+		- resource "aws_autoscaling_group" "bastion-privateweave-example-com" {
+		+   name = "bastions.bastions.privateweave.example.com"
+		-   name = "bastion.privateweave.example.com"
+		+   launch_configuration = "${aws_launch_configuration.bastions-bastions-privateweave-example-com.id}"
+		-   launch_configuration = "${aws_launch_configuration.bastion-privateweave-example-com.id}"
+		    max_size = 1
+		    min_size = 1
+		+   vpc_zone_identifier = ["${aws_subnet.utility-us-test-1a-privateweave-example-com.id}"]
+		-   vpc_zone_identifier = ["${aws_subnet.private-us-test-1a-privateweave-example-com.id}"]
+		    tag = {
+		      key = "KubernetesCluster"
+		...
+		    tag = {
+		      key = "Name"
+		+     value = "bastions.bastions.privateweave.example.com"
+		-     value = "bastion.privateweave.example.com"
+		      propagate_at_launch = true
+		    }
+		+   tag = {
+		+     key = "k8s.io/role/bastion"
+		+     value = "1"
+		+     propagate_at_launch = true
+		+   }
+		  }
+		  
+		...
+		    max_size = 1
+		    min_size = 1
+		+   vpc_zone_identifier = ["${aws_subnet.us-test-1a-privateweave-example-com.id}"]
+		-   vpc_zone_identifier = ["${aws_subnet.private-us-test-1a-privateweave-example-com.id}"]
+		    tag = {
+		      key = "KubernetesCluster"
+		...
+		    max_size = 2
+		    min_size = 2
+		+   vpc_zone_identifier = ["${aws_subnet.us-test-1a-privateweave-example-com.id}"]
+		-   vpc_zone_identifier = ["${aws_subnet.private-us-test-1a-privateweave-example-com.id}"]
+		    tag = {
+		      key = "KubernetesCluster"
+		...
+		    security_groups = ["${aws_security_group.bastion-elb-privateweave-example-com.id}"]
+		    subnets = ["${aws_subnet.utility-us-test-1a-privateweave-example-com.id}"]
+		+   health_check = {
+		+     target = "TCP:22"
+		+     healthy_threshold = 2
+		+     unhealthy_threshold = 2
+		+     interval = 10
+		+     timeout = 5
+		+   }
+		+   idle_timeout = 120
+		+ }
+		+ 
+		+ resource "aws_iam_instance_profile" "bastions-privateweave-example-com" {
+		+   name = "bastions.privateweave.example.com"
+		+   roles = ["${aws_iam_role.bastions-privateweave-example-com.name}"]
+		-   idle_timeout = 120
+		  }
+		  
+		...
+		  }
+		  
+		+ resource "aws_iam_role" "bastions-privateweave-example-com" {
+		+   name = "bastions.privateweave.example.com"
+		+   assume_role_policy = "${file("${path.module}/data/aws_iam_role_bastions.privateweave.example.com_policy")}"
+		+ }
+		+ 
+		  resource "aws_iam_role" "masters-privateweave-example-com" {
+		    name = "masters.privateweave.example.com"
+		...
+		  }
+		  
+		+ resource "aws_iam_role_policy" "bastions-privateweave-example-com" {
```

Comment:
Is this policy configurable? This is something else that has come up security wise.